src/libraries/Microsoft.PowerFx.Connectors/Public/ConnectorTableType.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    internal class ConnectorTableType : FormulaType
```

Comment:
>FormulaType [](http://example.com/codeflow?start=40&length=11)

can this derive from TableType?

Because of all the `Is TableType` checks.

Reply:
That suggests adding another TableType may really break some invariants. 

Why do we need this? Is existing TableType not sufficient?

Reply:
See TabularIRVisitor where we test this type
I could derive from (unsealed) TableType